### PR TITLE
Mh medic 2.0

### PR DESCRIPTION
**Pull request by bleehu**
Converting Medic to the 2.0 system. 
On reflection, another pass through medic should be made to make it more polished.

### CODE REVIEW

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 1]
```diff
@@ -39,36 +39,38 @@ Adrenaline Cost:30
 Needle Cost:	1
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: line 37, I think "Harmasist" should be "Harmacist", correct me if i'm wrong.

File: play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 2]
```diff
@@ -39,36 +39,38 @@ Adrenaline Cost:30
...
 Duration: 	2 Actions
 Range:		40m
-Effect:		10 % chance to miss. Deal 52 piercing damage.
-If target has no shield left and is still alive, apply a 
+Effect:		2 chance to miss. Deal 52 piercing damage. AP level 3.
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: Were you going to remove the "piercing" part of the damage from the medic abilities? I think it will help keep the players from being confused.

**Review by 1sourcecontrol — CHANGES_REQUESTED** [review]
Just two comments.. Looks great! I see that you nerfed a lot of your abilities, I haven't done any balance checks myself, so I'll be trusting your judgement there.

**Comment by bleehu** [comment]
Will make the changes. 
I might need to do another polishing pass to check their balance overall. A lot of the game has changed since I made this class...

File: CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 3]
```diff
@@ -50,7 +50,7 @@ Adrenaline Cost:30
 Needle Cost:	1
 Duration: 	1 Action
 Range:		40m
-Effect:		16 % chance to miss. Deal 52 damage.
+Effect:		2 chance to miss. Deal 52 damage. AP level 3
 If target has no shield left and is still alive, apply a 
 poison payload of your choice.
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: Fast Spike is purely better than Harmacist Spike, is that intentional?

Also, why does the text say 'no shield left' vs 'if pierces armor'? 
There's a couple of questions this brings... Normally conventional weapons are not stopped by most shields, so why does this reference shields when it isn't blocked by shields?
And is your intention that the needles will always apply poison regardless of piercing armor or not (because that's how it reads to me)?

File: CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 4]
```diff
@@ -60,17 +60,18 @@ Needle Cost: 3
 Duration: 	2 Actions
 Range:		30m
 Effect:		You launch off three harmasist spikes in quick succession using both
-hands. Each has an 18 % chance to miss. You may pick up to three targets to 
-launch at. Each spike that hits deals 52 damage. If target has no 
-shield left and is still alive, apply a poison payload of your choice.
+hands. Each has an 2 chance to miss. You may pick up to three targets to
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: >Each has an 2 chance to miss.

an -> a

File: CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 5]
```diff
@@ -60,17 +60,18 @@ Needle Cost: 3
...
-----Peircing Spike	-----
+----Piercing Spike	-----
 Adrenaline Cost:35
 Needle Cost: 1
 Duration: 	2 Actions
 Range:		45m
-Effect:		10 % chance to miss. Deal 42 damage.
-ignores shields and ballistic armor. Apply a poison payload of your choice.
+Effect:		2 chance to miss. Deal 42 piercing damage. AP level 6.
+if the  needle defeats the target's armor, apply a poison payload of your choice.
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: this doesn't say 'If target has no shield left," is that an intentional difference?
And it should say "pierces the target's armor" or "damages the targets health" because 'defeating' can be ambiguous: do you mean pierce the armor or go all the way through the armor points pool so that you damage health?

File: CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 6]
```diff
@@ -250,7 +251,8 @@ Duration: 	almost instantaneous
 Range:		Payload (use in combination with other powers)
 Effect:		A biologic target afflicted with this poison takes 180 damage that 
 ignores shields, silences the target and prevents them from casting nanite 
-spells for 10 turns if they survive.
+spells for 10 turns if they survive. (Also prevents engineering processes and
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: Should you change 

>nanite spells

to "Nanite Abilities"? Since we are moving away from the DnD 'spells' aesthetic

**Review by 1sourcecontrol — CHANGES_REQUESTED** [review]
In #228 you mentioned that you would catch the 'nanite' terms in this PR, but I don't see any changes? (For reference, https://github.com/bleehu/Compound_X/blob/95bf27f3ae4d36029353dc83c4a39cb877352280/CharacterCreation/Class-Specific%20Documentation/Medic%20Proceedures.txt#L267)

File: CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt (hunk 3 above)
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: It is an intentional improvement. I should probably have Harmacist spike improve at level 5 to offer a difference in kind.

It is my intention that if the needle does not defeat armor, that no poison is delivered. I will adjust the language.

File: CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt (hunk 4 above)
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: fixed in subsequent commit

File: CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 7]
```diff
@@ -564,36 +567,36 @@ Every target revealed by Sense Heartbeat that is communicated to an ally and is
...
 Effect:			Target gets a 30 point capacitive shield for the duration of 
 the engagement. This trick works on any race of ally. 
 
-----Nanite Restore	-----
+==Nanite Restore ==
 Adrenaline Reward:	n/4
 Range:			Touch
 Action:			Free Action
 Effect:			Transfer n number of Adrenaline points from the Doc to the 
 patient. If This trick is used on another doctor, no adrenaline points are earned.
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: >Transfer n number of Adrenaline points from the Doc to the 
  patient. If This trick is used on another doctor, no adrenaline points are earned. 

The name doesn't line up with the effect, anymore. Nanite Restore looks more like "Adrenaline Restore," and it doesn't make any sense to transfer adrenaline points to the patient if the patient isn't a doctor.

File: CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 8]
```diff
@@ -19,60 +19,67 @@ The net effect is that a medic starts every engagement with 100 adrenaline
...
-----Harmacist Spike	-----
+==Harmacist Spike ==
 Adrenaline Cost:30
 Needle Cost:	1
 Duration: 	2 Actions
 Range:		40m
-Effect:		10 % chance to miss. Deal 52 damage.
-If target has no shield left and is still alive, apply a 
-poison payload of your choice.
+Effect:		2 chance to miss. Deal 52 damage. AP level 3. If the spike deals
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: >Deal 52 damage.

Should this say "piercing damage"?

File: CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 9]
```diff
@@ -19,60 +19,67 @@ The net effect is that a medic starts every engagement with 100 adrenaline
...
-----Fast Spike	-----
+==Fast Spike ==
 Adrenaline Cost:30
 Needle Cost:	1
 Duration: 	1 Action
 Range:		40m
-Effect:		16 % chance to miss. Deal 52 damage.
-If target has no shield left and is still alive, apply a 
-poison payload of your choice.
+Effect:		2 chance to miss. Deal 52 damage. AP level 3. If the spike deals
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: >Deal 52 damage.

Should this say "piercing damage"?

File: CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 10]
```diff
@@ -243,16 +250,17 @@ exposed biology are affected by this even if a deployment method says they
...
 regions of space.
 
-----Poison:	Lethal Toxin	-----
+==Poison:	Lethal Toxin ==
 Adrenaline Cost: 60
 Contagion Cost: 1
 Duration: 	almost instantaneous
 Range:		Payload (use in combination with other powers)
 Effect:		A biologic target afflicted with this poison takes 180 damage that 
 ignores shields, silences the target and prevents them from casting nanite
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: I missed mentioning another thing---

>prevents them from casting nanite 

Should probably say something like "prevent them from _using_ Nanite"...
Since we don't 'cast ablities/spells' anymore.

File: CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 11]
```diff
@@ -19,60 +19,67 @@ The net effect is that a medic starts every engagement with 100 adrenaline
...
 Adrenaline Cost: 40
 Needle Cost: 3
 Duration: 	2 Actions
 Range:		30m
 Effect:		You launch off three harmasist spikes in quick succession using both
-hands. Each has an 18 % chance to miss. You may pick up to three targets to 
-launch at. Each spike that hits deals 52 damage. If target has no 
-shield left and is still alive, apply a poison payload of your choice.
+hands. Each has a 2 chance to miss. You may pick up to three targets to 
+launch at. Each spike that hits deals 52 piercing damage and has an armor piercing
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: armor piercing -> Armor Piercing
or -> AP

File: CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 12]
```diff
@@ -19,60 +19,67 @@ The net effect is that a medic starts every engagement with 100 adrenaline
...
 Adrenaline Cost:30
 Needle Cost:	1
 Duration: 	2 Actions
 Range:		40m
-Effect:		10 % chance to miss. Deal 52 damage.
-If target has no shield left and is still alive, apply a 
-poison payload of your choice.
+Effect:		2 chance to miss. Deal 52 damage. AP level 3. If the spike deals
+damage to the target's health and the target is still alive, apply a poison 
+payload of your choice. At Player level 5, the Harmacist spike deals 64 damage
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: piercing damage?

File: CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt [hunk 13]
```diff
@@ -264,32 +272,26 @@ recursive without limit, but a target that has made a save against the
...
+==Free Running ==
 Adrenaline Cost: 60
 Duration: 	2 Actions
 Range:		5m radius centered at a point up to 30m from the Doc
 Effect:		Your nanites form microscopic bonds with any solid surface allowing 
 movement at your speed along any flat, solid surface that can support your 
 weight. The procedure for doing this without blowing up one's heart or leaving
-one's spine behind takes more training than the average soldier receives. 
+one's spine behind takes more training than the average soldier receives. It 
+also can be quite exhausting.
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt]: Free Running is so cool! Makes me want to play medic, just to get this ability.

**Review by 1sourcecontrol — CHANGES_REQUESTED** [review]
I'm trying to stay ahead of the game! Good work so far!

**Comment by bleehu** [comment]
Thanks for the encouraging comments. Feels like you tore me a new one. But it was necessary. As it is, Medic is off of its aesthetic, and the hodge-podge of old and new wasn't given the level of detail that it needed. I'll handle Issue #234 in this pull as well. It's time to fix my stuff as they say! :)

**Comment by 1sourcecontrol** [comment]
Sure thing! I'm trying to mention my good thoughts, more. One of the down sides to our online source control, it can feel really cold if we only list what's essential for a fix -without giving our thoughts on what we do like.
I think you're doing a great job fixing this up, and making the class in the first place. There's an incredible amount of content in the engineer and doctor/medic classes.

**Comment by bleehu** [comment]
Made a bunch of changes. Still need to replace
[ ] Nanite Restore
and add
[ ] whateverectomy
There may yet be more issues, but I need to catch a train.

**Comment by bleehu** [comment]
Can I get a "fix this moar"? ;)

**Comment by 1sourcecontrol** [comment]
WEll, I mean, If you really want me to..

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt [hunk 14]
```diff
@@ -0,0 +1,718 @@
...
+
+    Their Eye: The target loses one of their eyes. They take a permanent -2 to
+their Perception and receive 60 damage. Their to hit score is reduced by 2. 
+You gain 1 Contagion.
+
+    Their Spleen: The target loses their spleen. They take 60 damage and their
+Fortitude is permanently reduced by 2. You gain 2 Contagions.
+
+    Their Lymph Node: The target loses their lymph node. They lose 40 health and
+120 Nanites. You gain 2 Contagions.
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: >and 120 Nanites

Can we add an "up to" or something? To be crystal clear that the player won't get negative Nanites if they have a pool of 80, for example.

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt [hunk 15]
```diff
@@ -0,0 +1,718 @@
...
+Action:			4 Actions
+    Stands for Persistent Emergent Generalists, a type of surgical nanobot that 
+stays in the patient's bloodstream and continues to administer care over an 
+extended period of time. Effect lasts for 10 turns and heals 10 health every 
+turn. DC 60 to hack. If nanobots are hacked, deals damage instead. PEGs are 
+easily detectable by tech scanning technologies or spells. PEGs cannot heal a 
+patient over their maximum health.
+
+==Improved P.E.G.s ==
+Adrenaline Reward:	50 Nanites
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: >Adrenaline Reward:	50 Nanites

This isn't technically an Adrenaline reward, is it?

Also, are we strictly keeping Nanites out of doctor "spells" still? (Confirming in order to help me avoid false positives in my debug.)

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt [hunk 16]
```diff
@@ -0,0 +1,718 @@
...
+Effect:			Target gets a 30 point capacitive shield for the duration of 
+the engagement. This trick works on any race of ally. 
+
+==Nanite Restore ==
+Adrenaline Reward:	n/4
+Range:			Touch
+Action:			Free Action
+Effect:			Transfer n number of Adrenaline points from the Doc to the 
+patient. If This trick is used on another doctor, no adrenaline points are 
+earned.
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:710]: Still have "Nanite Restore" here, restoring adrenaline points from doc to x patient.

If the intention is to give the target X Nanites for X adrenaline points (1 for 1) then I'd need some clarification in the description.

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt [hunk 17]
```diff
@@ -0,0 +1,718 @@
...
+are in the affected area when the Doctor sprays it or that end their turn in the
+area for 5 turns after it is sprayed must make a FORT save. The FORT save has a
+DC of 110. If a target succeeds on the save the area does not affect them and
+they do not need to save again if they are still in the area on subsequent 
+turns. If they fail the save, they are "contaminated." Races that are extra
+sensitive to smoke or disease have disadvantage on this save. At level 10, this
+area expands to up to 3m x 5m at the discretion of the Doctor.
+
+    "Contaminated" targets get a -2 to melee or weapon accuracy rolls, and lose
+5 health and 5 nanites at the end of their turn every turn until Contamination
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: Idk if we have rules on it, but what happens when the target has 0 Nanites in their Nanite pool? Or what if they had 3 Nanites and need to lose 5?

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt [hunk 18]
```diff
@@ -0,0 +1,718 @@
...
+
+==Poison:	Lethal Toxin ==
+Adrenaline Cost: 60 Points
+Contagion Cost: 1 Contagion
+Duration: 	almost instantaneous
+Range:		Payload (use in combination with other powers)
+    A biologic target afflicted with this poison must succeed on a DC 110 FORT 
+save or the target takes 180 damage that ignores shields and armor and silences 
+the target. If the target succeeds on their save, they take 60 damage instead. A 
+silenced target is prevented from using nanite abilities for 10 turns. (Also
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: "nanite" -> "Nanite"

We might want to change the text to say "active abilities that use Nanites" because, strictly speaking, we never formally tag an ability as a "Nanite Ability". Just a thought.

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt [hunk 19]
```diff
@@ -0,0 +1,718 @@
...
+Range: Applies to the Doc only
+    For 3 turns, the doc gets a -3 to miss with guns, grenades, spells, and 
+melee weapons. They also get a +30 to reload and a +1 chance to crit.
+
+==Free Running ==
+Adrenaline Cost: 60 Points
+Nanite Cost: 15 Nanites
+Duration: 	2 Actions
+Range:		5m radius centered at a point up to 30m from the Doc
+    Your nanites form microscopic bonds with any solid surface allowing
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: "nanites" -> "Nanites"

FYI I've been capitalizing "nanite/nanites" throughout all of the CX rules. But I'm unsure if we actually do need to capitalize it. I've been treating it like a proper noun, but come to think of it, I don't know if it is. I don't think there is a dictionary definition of "nanite/nanites" yet.

**Review by 1sourcecontrol — CHANGES_REQUESTED** [review]
Some Nanite'y things

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt [hunk 20]
```diff
@@ -0,0 +1,718 @@
...
+Adrenaline Cost: 60 Points
+Contagion Cost: 1 Contagions
+Duration: 1 Action
+Range: Applies to the Doc only
+    For 3 turns, the doc gets a -3 to miss with guns, grenades, spells, and 
+melee weapons. They also get a +30 to reload and a +1 chance to crit.
+
+==Free Running ==
+Adrenaline Cost: 60 Points
+Nanite Cost: 15 Nanites
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:359]: Can I get your thoughts on this ability? Just curious about the re-introduction of a Nanite Cost.

Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:359]: I kinda like the idea of enabling the medic to get to their patients very easily, but I think this ability requires the deployment of nanite technology. Specifically nanite technology over the medic's medical technology. What do you think?

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt (hunk 16 above)
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:710]: I need to replace this with something, I'm just not sure what to replace it with.

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt (hunk 15 above)
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: Wow. I can't believe I missed this. Thank you for catching it!

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt (hunk 14 above)
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: Absolutely! Will do! good catch.

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt (hunk 19 above)
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: Good Call. Will Ctrl+H it.

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt (hunk 18 above)
Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: Good clarification.

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt (hunk 20 above)
Comment (Turtlelord26) [reply CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:359]: We really need to have that formal design meeting where we hash out once and for all exactly what our resource bar is, how it works, and what position nanites have in the new aesthetic.

Comment (1sourcecontrol) [reply CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:359]: Yeah, I guess I'm not sure what to think until we formalize nanite lore. 

My gut says adding Nanites to this specific ability is as good as not having the Nanites cost there at all, if the doctor never uses other nanite abilities. Their pool might be completely unused. Other than that, seems like a good restriction and lore friendly (well, trowl223 era lore friendly).

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt [hunk 21]
```diff
@@ -325,9 +327,10 @@ Range:		Payload (use in combination with other powers)
     A biologic target afflicted with this poison must succeed on a DC 110 FORT 
 save or the target takes 180 damage that ignores shields and armor and silences 
 the target. If the target succeeds on their save, they take 60 damage instead. A 
-silenced target is prevented from using nanite abilities for 10 turns. (Also 
-prevents Engineering Processes and Doctor Procedures due to violent tremors). At 
-level 13, damage increases to 220.
+silenced target is prevented from using abilities that use Nanites for 10 turns.
```
Comment (1sourcecontrol) [inline CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: We didn't add silenced to the Glossary yet, did we?
Regardless, I think "silences the target" should be changed because the target isn't speaking spells (anymore). Can you rephrase the text to remove or rephrase  'silences'/'silenced'?

**Review by 1sourcecontrol — CHANGES_REQUESTED** [review]

File: CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt (hunk 21 above)
Comment (Turtlelord26) [reply CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: "Silence" does sound out of aesthetic, but we should definitely have a word for the state of not being able to use our nanite abilities. How about Suppressed?

Comment (1sourcecontrol) [reply CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: "Suppressed" is already taken for being under the effects of Light/Med/Heavy Suppression by a typical ranged weapon. Although I think Suppressed would be a great fit here, as well.

What about "Disrupted"? (Does that have overlap with hacking/robot disabling of some kind?)

Or maybe some term that signifies the inability for the target to calm their arms/physical nerves...?

Comment (1sourcecontrol) [reply CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: P.S. I'm not sure if "Suppressed" is added to the glossary yet

Comment (Turtlelord26) [reply CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: Totally forgot about suppression levels :(
How about Stifled?

Comment (bleehu) [reply CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt]: This is the only part of the game (as far as I know) where the Silence effect is used. I don't know that it makes sense to encapsulate it as a status effect? Would other classes/enemies/hazards use a similar Condition?  

If we can make it more aesthetically intuitive by renaming it, we should do that. The reason for the original name of "silenced" comes from the standard industry term for such abilities as used in WoW, LoL, CoH, etc where the term is used to mean "cannot cast spells". Tactically, I think such an ability makes a lot of sense for the medic to have, and I would like to update/polish the ability until it makes sense and works well rather than yanking it. 
Looking forward to more feedback. If I'm understanding this right, this is the last blocker to updating the Doctor class?

**Comment by bleehu** [comment]
Mentioning Issue #242 here because merging this fixes it.

**Comment by bleehu** [comment]
@Turtlelord26 after 5 rounds of changes requested over a month, I fear this particular pull request has suffered the worst kind of scope creep. If this playable and technically in the combat 2.0 system can we merge it and address any other issues in a more atomic manner?

**Review by Turtlelord26 — APPROVED** [review]
Yes, playable. Non-conflicting with c2.0 systems.
